packages/server/graphql/types/TeamPromptResponse.ts

```diff
@@ -0,0 +1,45 @@
+import {GraphQLFloat, GraphQLID, GraphQLNonNull, GraphQLObjectType, GraphQLString} from 'graphql'
+import {GQLContext} from '../graphql'
+import Reactable, {reactableFields} from './Reactable'
+import Team from './Team'
+
+const TeamPromptResponse: GraphQLObjectType = new GraphQLObjectType<any, GQLContext>({
+  name: 'TeamPromptResponse',
+  description: 'A response of a single team member in a team prompt',
+  interfaces: () => [Reactable],
+  fields: () => ({
+    ...reactableFields(),
```

Comment:
I've added a reactable fields here but I guess it'd be best to create a task for the implementation of that. DB has the list of reactjis for each response but I'm not sure if the data format matches 100%.